src/components/inputCheckbox/AposCheckbox.vue

```diff
@@ -36,7 +37,16 @@ export default {
     CheckBoldIcon,
     MinusIcon
   },
+  // Custom model to handle the v-model connection on the parent.
+  model: {
```

Comment:
We're telling Vue to replace the default `v-model` model. https://vuejs.org/v2/api/#model